renderer/index.js

```diff
@@ -795,6 +799,44 @@ function toggleSelectTorrent (infoHash) {
   update()
 }
 
+function openTorrentContextMenu (torrentSummary) {
+  var menu = new remote.Menu()
+  menu.append(new remote.MenuItem({
+    label: 'Save Torrent File As...',
+    click: () => saveTorrentFileAs(torrentSummary)
+  }))
+
+  menu.append(new remote.MenuItem({
+    label: 'Copy Instant.io Link to Clipboard',
+    click: () => clipboard.writeText(`https://instant.io/#${torrentSummary.infoHash}`)
+  }))
+
+  menu.append(new remote.MenuItem({
+    label: 'Copy Magnet Link to Clipboard',
+    click: () => clipboard.writeText(torrentSummary.magnetURI)
+  }))
+
+  menu.popup(remote.getCurrentWindow())
+}
+
+function saveTorrentFileAs (torrentSummary) {
+  var newFileName = `${path.parse(torrentSummary.name).name}.torrent`
+  var opts = {
+    title: 'Save Torrent File',
+    defaultPath: path.join(state.saved.downloadPath, newFileName),
+    filters: [{ name: 'Torrents', extensions: ['torrent'] }]
```

Comment:
Actually, Windows programs seem to default to the same file type as the program. Looks good to me, then.

Will add an All Files option though, since programs seem to have those.